laske_export/document/invoice_sales_order_adapter.py

```diff
@@ -155,12 +139,20 @@ def get_line_items(self):
         for i, invoice_row in enumerate(invoice_rows):
             line_item = LineItem()
 
-            if invoice_row.receivable_type == self.receivable_type_rent:
+            receivable_type = invoice_row.receivable_type
+            # When dealing with rent invoice rows, we look up the SAP codes from the LeaseType object...
+            if receivable_type == self.receivable_type_rent:
                 line_item.material = self.invoice.lease.type.sap_material_code
                 line_item.order_item_number = self.invoice.lease.type.sap_order_item_number
+
+            # ...but in other cases the SAP codes are found in the ReceivableType object of the invoice row.
+            elif receivable_type == ReceivableType.objects.get(pk=8):
```

Comment:
You could handle this hard coded receivable type the same way as the "self.receivable_type_rent". I.e. fetched in the exporter and passed to the class.

Reply:
Good point. Will change.